frontend/app/components/wp-table/timeline/wp-timeline-global.directive.ts

```diff
@@ -98,8 +98,12 @@ export class WpTimelineGlobalService {
 
       // start
       const startCell = this.cells[e.from];
-      let lastX = startCell.getRightmostPosition();
 
+      if (!startCell) {
```

Comment:
/cc @romanroe This caused trouble on the timeline branch, so for sake of testing the merge with hierarchies, i've added this simple check. Please revise